common/src/main/java/com/wynntils/models/territories/GuildAttackTimerModel.java

```diff
@@ -74,11 +75,11 @@ public GuildAttackTimerModel(MarkerModel marker) {
     public void onMessage(ChatMessageReceivedEvent event) {
         if (event.getRecipientType() != RecipientType.GUILD) return;
 
-        Matcher matcher = event.getOriginalStyledText().getMatcher(WAR_MESSAGE_PATTERN);
+        Matcher matcher = event.getStyledText().getMatcher(WAR_MESSAGE_PATTERN);
```

Comment:
You don't seem to use the "stripAlignment" method here? Also, watch out for double whitespaces, that are after those unicode texts, you may need to do a `replaceAll("\s\s", "")`